documentation/docs/pages/developers/nymvpncli.mdx

```diff
@@ -0,0 +1,278 @@
+import { Callout } from 'nextra/components'
+
+# Nym VPN CLI
+
+This is a short guide to setting up and using the `nym-vpnc` tool, which is used in conjunction with the `nym-vpnd` daemon.
+
+<Callout type="warning" emoji="⚠️">
+  These binaries have superceded the older `nym-vpn-cli` binary. This still operates for the moment as it is being used in testing scenarios but will go out of date quickly.
+</Callout>
+
+Download and run instructions for the GUIs can be found [here](https://nymvpn.com/en/download/linux).
+
+## Download & Extract Binary
+Check the [release page](https://github.com/nymtech/nym-vpn-client/releases/) page for the latest release version and modify the instructions accordingly. These instructions use the latest as of the time of writing.
+```sh
+wget -q https://github.com/nymtech/nym-vpn-client/releases/download/nym-vpn-core-v1.1.0-beta.3/nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
+tar -xzf nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>.tar.gz &&
+cd nym-vpn-core-v1.1.0-beta.3_<YOUR_OPERATING_SYSTEM>/ &&
+chmod u+x *
+```
+
+## Build from Source
+### Prerequisites
+All operating systems require both [Rust](https://www.rust-lang.org/tools/install) and [Go](https://go.dev/doc/install).
+
+**Arch specific packages:**
+```sh
+yay -S gcc make protobuf base-devel clang
+```
+
+**Ubuntu24 specific packages:**
+```sh
+apt install gcc make protobuf-compiler pkconfig libdbus-1-dev build-essential clang
+```
+
+<Callout type="warning" emoji="⚠️">
+  Older Debian/Ubuntu versions need to manually install `protobuf-compiler` >= v3.21.12
+</Callout>
+
+### Clone & `make`
+```sh
+git clone https://github.com/nymtech/nym-vpn-client.git
+cd nym-vpn-client/
+make
+```
+
+## Mnemonic Generation
+Create an account at [nymvpn.com](nymvpn.com) to obtain your mnemonic.
+
+## Start the daemon
+```sh
+sudo ./PATH/TO/nym-vpnd
```

Comment:
I would consider to:
```
import { VarInfo } from 'components/variable-info.tsx';
```

Then put 
```
<VarInfo />
```
Somewhere on top and use the PATH cnvention like it's noted there. 

https://nymtech.net/docs/operators/variables

Which is `<PATH_TO>`